pkg/platform/provider/baremetal/phases/kubelet/kubelet.go

```diff
@@ -84,3 +84,21 @@ func Install(s ssh.Interface, version string) (err error) {
 
 	return nil
 }
+
+func StopService(s ssh.Interface) (err error) {
```

Comment:
marge StopService StartService to one function,  the function support start/stop/restart